ESCPOS_NET.ConsoleTest/Test2DCodes.cs

```diff
@@ -25,7 +25,7 @@ public static partial class Tests
             e.PrintLine(),
 
             e.PrintLine("QRCODE MODEL 2:"),
-            e.Print2DCode(TwoDimensionCodeType.QRCODE_MODEL2, websiteString),
+            e.Print(websiteString),
```

Comment:
I think this should be PrintQRCode

Reply:
yep yep yep. Sorry about that, missed it.

Reply:
Fixed in 8abffce68f711bf4128c207407e49e5e1fe01b1b